str/trtools/merge_str_prep.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+# pylint: disable=duplicate-code
+"""
+This script prepares GangSTR/EH VCF files for input into mergeSTR. 
+Required input: --caller, --input-dir, and external sample IDs
+For example: 
+analysis-runner --access-level test --dataset tob-wgs --description 'tester --output-dir 'tester' merge_prep.py --caller=eh --input-dir=gs://cpg-tob-wgs-main/str/expansionhunter/pure_repeats --dataset=tob-wgs TOBXXXX TOBXXXX
+
+Required packages: sample-metadata, hail, click, os
+pip install sample-metadata hail click
+
+"""
+import os
+import click
+import hailtop.batch as hb
+
+from sample_metadata.apis import SampleApi
+
+from cpg_utils.config import get_config
+from cpg_utils.hail_batch import remote_tmpdir, output_path
+
+config = get_config()
+
+REF_FASTA = 'gs://cpg-common-main/references/hg38/v0/Homo_sapiens_assembly38.fasta'
+BCFTOOLS_IMAGE = config['images']['bcftools']
+
+
+# inputs:
+# caller
+@click.option('--caller', help='gangstr or eh')
+# dataset
+@click.option('--dataset', help='dataset eg tob-wgs')
+# input directory
+@click.option('--input-dir', help='gs://...')
+# input sample ID
+@click.argument('external-wgs-ids', nargs=-1)
+@click.command()
+def main(
+    dataset, caller, input_dir, external_wgs_ids: list[str]
+):  # pylint: disable=missing-function-docstring
+
+    # Initializing Batch
+    backend = hb.ServiceBackend(
+        billing_project=get_config()['hail']['billing_project'],
+        remote_tmpdir=remote_tmpdir(),
+    )
+    b = hb.Batch(backend=backend, default_image=os.getenv('DRIVER_IMAGE'))
```

Comment:
similar to above, you can sub this out for `get_batch` and save yourself a few lines